src/main/java/io/openliberty/tools/common/plugins/util/AbstractContainerSupportUtil.java

```diff
@@ -96,8 +110,11 @@ private boolean checkPodmanVersion() throws PluginExecutionException  {
         if (VersionUtility.compareArtifactVersion(podmanVersion, MIN_PODMAN_VERSION, false) < 0) {
             throw new PluginExecutionException("The detected Podman client version number is not supported:" + podmanVersion.trim() + ". Podman version must be " + MIN_PODMAN_VERSION + " or higher.");
         }
-        isDocker = false;
-        checkedContainerType = true;
+
+        if (!checkedContainerType) {
+            isDocker = false;
```

Comment:
I feel like setting `isDocker` to false might be needed even when `checkedContainerType` is true. In the case that `setIsDocker` is called with the parameter set to `true` and Docker is not installed, the code will call `checkPodmanVersion` as the fallback.

Reply:
Should we skip the Podman check if `checkedContainerType` is true? If there's no Docker, but a valid Podman version, do we want to use Podman even though it was set to use Docker?